PA#1Herve.ASM

```diff
@@ -0,0 +1,103 @@
+# COMP3410 Program Template
+# Author: Herve Aniglo
+# Assignment: PA1
+# Date: 02/10/2015
+
+# Turn in one .asm file per assignment component
+# Remember to submit it as a pull request to the GitHub repo for the assignment
+
+# fibonacci.asm
+# Input: None
+# Output: Print to terminal N Fibonacci numbers
+
+##########################################################
+# .data segment. Constant and variable definitions go here.
+##########################################################
+
+	.data
+
+fibs:	.word   0 : 9         # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
+size:	.word  9              # create a single integer variable named "size" that indicates the length of the array
+text:	.asciiz "Enter a number 1-9: "
+
+###############################################
+# .text segment. Assembly instructions go here.
+###############################################
+	.text
+	      la   $s0, fibs        # load address of array into $s0
+	      #la   $s5, size        # load address of size variable into $s5
+	      #lw   $s5, 0($s5)      # load array size from its address in the register
+
+#Check to make sure user does not exceed 9 or 0	      
+check:		li $v0, 4		# 4 = print string
+	      la $a0, text	# load the address of text
+	      syscall		# do the printing
+	      
+	      li $v0, 5		# 5 = read int
+	      syscall		# read the int, result stored in $v0
+	      move $s5, $v0
+	      
+	bgt $v0, 9, done1
```

Comment:
I suspect that you intended to jump back to the "check:" label if line 40, if $v0 > 9 evaluates to true in order to get input again, but since it points to done1 that line doesn't really do anything.  If that statement is true, it goes to line 41.  If that statement is true it goes to line 41